backend/src/security/permissions.js

```diff
@@ -1,43 +1,29 @@
-const { isAuthenticated, isPostUpvoted, isPostWithTitlePresent, passwordIsValid, passwordIsTooShort, emailIsTaken, canSeeEmail} = require("./rules");
-const { shield, and, not, deny, allow,  chain} = require('graphql-shield');
-const { UserInputError } = require('apollo-server');
+const { isAuthenticated, canSeeEmail } = require("./rules");
+const { shield, and, not, deny, allow} = require('graphql-shield');
 
 const permissions = shield({
     Query: {
-      "*": allow,
+      "*": deny,
```

Comment:
:star: You deny the client to call mutations and queries of your subschema directly.